x/ibc/scripts/init-allorad.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+BINARY=./allorad
+CHAIN_DIR=./data
+CHAINID=allora_demo
+RLY_MNEMONIC="alley afraid soup fall idea toss can goose become valve initial strong forward bright dish figure check leopard decide warfare hub unusual join cart"
```

Comment:
I don't know how great an idea it is to store mnemonics in code, even for testing purposes, can we can do something better here?

Reply:
e.g. create a new key like alice or bob and then use that instead of hardcoding a mnemonic